feat(support): add cachy browser migration guide (#132)
CachyOS/wiki: src/content/docs/support/faq.mdx

Cachy Browser is based on Firefox (via LibreWolf), so migrating your profile, including history, bookmarks, extensions, and settings, to the standard Firefox is generally possible by copying the profile directory. However, always back up your data before proceeding.

There are a few methods, ranging from a full profile copy to using sync or manual transfer.

**Important First Step: Back Up Your Profiles!**

Before you start, it's crucial to back up both your Cachy Browser profile and your Firefox profile (if you have existing data you want to keep). This way, you can restore them if something goes wrong. Simply copy the entire profile folders (locations described below) to a safe place.

#### Method 1: Direct Profile Copy

This method attempts to copy *everything* from your Cachy Browser profile to Firefox. It's the most likely way to transfer all settings and extension data but carries a small risk of minor incompatibilities.

1.  **Locate Your Cachy Browser Profile Folder:**
    *   Open Cachy Browser.
    *   Type `about:profiles` in the address bar and press Enter.
    *   Find the profile currently in use (it usually says "This is the profile in use...").

>     *   Click the "Open Directory" (or similar wording like "Show in Finder" on macOS, "Open Folder" on Windows/Linux) button next to the "Root Directory". This will open the folder containing your profile data.

Reviewer: lets keep only "Open Directory", as thats what Cachy Browser has on CachyOS

no need for "Show in Finder" and "Show in Finder"

>     *   Alternatively, you can use `about:support` and find the "Profile Folder" entry.

Reviewer: ```suggestion
    *   Alternatively, you can use `about:support` and find the "Profile Directory" entry.
```

>     *   **Typical Location on Linux:** `~/.cachy-browser/cachy-browser/<profile_name>/` or sometimes `~/.mozilla/firefox/<profile_name>/` if it used standard paths. The `<profile_name>` is usually a random string followed by `.default` or similar.

Reviewer: ```suggestion
    *   **Typical Location:** `~/.cachy/<profile_name>/`. The `<profile_name>` is usually a random string followed by `.default` or similar.
```

    *   Click its "Open Directory" button.
    *   **Typical Location on Linux:** `~/.mozilla/firefox/<profile_name>/`

3.  **Close Both Browsers:** Ensure Cachy Browser and Firefox are completely closed.

4.  **Copy and Replace:**
    *   Go to the Cachy Browser profile folder you opened in Step 1. Select *all* the files and folders inside it (`Ctrl+A`).
    *   Copy these files (`Ctrl+C`).
    *   Go to the Firefox profile folder you opened in Step 2.
    *   **Delete** all the existing files and folders inside the Firefox profile folder. (This is why the backup is important!).
    *   Paste the files you copied from the Cachy Browser profile folder (`Ctrl+V`).

5.  **Launch Firefox and Verify:**
    *   Start Firefox. It should now load with your Cachy Browser bookmarks, history, extensions, and settings.
    *   Check your bookmarks, history, and especially your extensions.

6.  **Troubleshooting:**
    *   **Extensions:** Some extensions might require reconfiguration or might not work if they were specific Cachy Browser versions or relied on features not in standard Firefox. You may need to reinstall them from the official Firefox Add-ons store.

>     *   **Privacy Settings:** Cachy Browser likely had stricter privacy settings enabled by default (similar to LibreWolf). Review Firefox's privacy and security settings (`about:preferences#privacy`) and adjust them to your liking.

Reviewer: ```suggestion
    *   **Privacy Settings:** Cachy Browser had stricter privacy settings enabled by default (similar to LibreWolf). Review Firefox's privacy and security settings (`about:preferences#privacy`) and adjust them to your liking.
```

    *   Choose the data you want to sync (Bookmarks, History, Passwords, Extensions, Settings, Open Tabs).
    *   Allow Cachy Browser time to fully sync the data to the server.

2.  **Enable Sync in Firefox:**
    *   Open Firefox.
    *   Go to Settings (`about:preferences`) > Sync.
    *   Sign in with the *same* Firefox Account.
    *   Ensure the data types you want are selected for syncing.
    *   Allow Firefox time to download the synced data.

3.  **Limitations:** Firefox Sync is great for core data but often doesn't sync extension *settings* or *data*, and might not sync *all* browser preferences (`about:config` changes). You'll likely need to reconfigure extensions and some specific browser settings manually.

#### Method 3: Manual Data Transfer (Selective)

If the above methods fail or you only want specific data:

1.  **Bookmarks:**
    *   In Cachy Browser: Go to Bookmarks > Manage Bookmarks (or `Ctrl+Shift+O`). Click "Import and Backup" > "Export Bookmarks to HTML...". Save the file.
    *   In Firefox: Go to Bookmarks > Manage Bookmarks. Click "Import and Backup" > "Import Bookmarks from HTML...". Select the file you saved.
2.  **Passwords:**

>     *   In Cachy Browser: Go to Settings > Privacy & Security > Logins and Passwords > Saved Logins. Click the three-dots menu (...) and choose "Export Logins...". Save the `.csv` file (be aware this file is unencrypted).

Reviewer: ```suggestion
    *   In Cachy Browser: Go to Passwords (`about:logins`). Click the three-dots menu (...) and choose "Export Passwords...". Save the `.csv` file (be aware this file is unencrypted).
```

>     *   In Firefox: Go to Settings > Privacy & Security > Logins and Passwords > Saved Logins. Click the three-dots menu (...) and choose "Import from a File...". Select the `.csv` file. Delete the `.csv` file securely afterwards.

Reviewer: ```suggestion
    *   In Firefox: Go to Passwords (`about:logins`). Click the three-dots menu (...) and choose "Import from a File...". Select the `.csv` file. Delete the `.csv` file securely afterwards.
```
